2996_reflection/reflection.md

```diff
@@ -989,7 +993,9 @@ struct universal_formatter {
 
     template for (constexpr auto mem : nonstatic_data_members_of(^T)) {
       delim();
-      out = std::format_to(out, ".{}={}", name_of(mem), t.[:mem:]);
+      std::string_view mem_label = has_identifier(mem) ? identifier_of(mem)
+                                                       : "(unnamed-member)";
+      out = std::format_to(out, ".{}={}", mem_label, t.[:mem:]);
```

Comment:
This one I guess is more interesting since what would you actually want to _do_ in this case? But that's not our job.